CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### Jun 19, 2025
+- Document the use of image tags in openshift Deployments. [DESENG-841](https://citz-gdx.atlassian.net/browse/DESENG-841)
```

Comment:
Small detail, just a missing space between the date and the description of your change log entry.